lib/store/index.js

```diff
@@ -288,6 +289,21 @@ export class Store {
     this.kernelMapping.set(newKey, this.kernelMapping.get(oldKey));
     this.kernelMapping.delete(oldKey);
   }
+
+  enableAutocomplete(watchEditor: Function) {
+    this.addAutocomplete = watchEditor;
+    for (let kernel of this.runningKernels) {
+      kernel.watchesStore.addAutocomplete();
```

Comment:
I was thinking about making a store for other package consumption, which could separate hydrogen logic from package add-on logic since it is not essential to hydrogen running. Let me know what you think about that. 
As for the logic, if it doesn't live here I believe it would have to live in the `main.js` if we left everything else the same. The reason being is because there is no way to access the `watchesStore` without the `kernels` which a stored in the store. Like I was saying above though I could technically move it to a new file (store or no store) I just cant encapsulate all of the code in watchesStore.